### ISSUE TITLE
Font rendering looks weired on Linux

### ISSUE THREAD

**Issue by mrv90** [Microsoft/vscode#46022]
Issue Type: <b>Bug</b>

Font rendering is too bad ! It's so difficult and weired to read titles and text of vs code !

VS Code version: Code 1.21.1 (79b44aa704ce542d8ca4a3cc44cfca566e7720f1, 2018-03-14T14:46:30.761Z)
OS version: Linux x64 4.6.0-kali1-amd64

<details>
<summary>System Info</summary>

|Item|Value|
![screenshot from 2018-03-17 15-12-30](https://user-images.githubusercontent.com/2759185/37555015-427614ba-29f6-11e8-9733-2626a3f0a2a8.png)

|---|---|
|CPUs|Intel(R) Pentium(R) CPU G3240 @ 3.10GHz (2 x 3100)|
|Load (avg)|1, 1, 1|
|Memory (System)|7.70GB (2.53GB free)|
|Process Argv|/usr/share/code/code --unity-launch|
|Screen Reader|no|
|VM|0%|

</details>Extensions: none
Reproduces without extensions

**Comment by tsalinger** [comment]
Go to your user settings (CTRL+P + ,) and check what is set for the "editor.fontFamily" setting. It looks to me as if some weird font is in use.

**Comment by allywilson** [comment]
Same thing happening to me. Tried changing the font to something completely different (I used 'Hack' font to test), get the same result.

VS Code details:
Version 1.21.1
Commit 79b44aa704ce542d8ca4a3cc44cfca566e7720f1
Date 2018-03-14T14:46:30.761Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64

OS: Ubuntu 18.04 Beta. kernel: 4.15.0-12-generic

Image example: https://imgur.com/9Z8Idbr
